Traceback (most recent call last):
  File "<ipython-input-9-df23c12b1b57>", line 12, in <module>
    raise Exception('This is the erro message')#Gera a exception
Exception: This is the erro message
